core/src/main/java/bisq/core/util/PriceUtil.java

```diff
@@ -72,7 +72,8 @@ public static MonetaryValidator getPriceValidator(boolean isFiatCurrency) {
     }
 
     public static InputValidator.ValidationResult isTriggerPriceValid(String triggerPriceAsString,
-                                                                      Price price,
```

Comment:
I'd rater use the MarketPrice object exposing the currencyCode property instead of adding another parameter. WDYT?

Reply:
Good idea - [fixed](https://github.com/bisq-network/bisq/pull/6109/commits/55c55dbf472e14d4bea938469c1e54c9cd75df63)